addons/binding/org.openhab.binding.logreader/README.md

```diff
@@ -0,0 +1,107 @@
+
+## Channels
+
+List of channels
+
+| Channel Type ID    | Item Type    | Description                                                    |
+| ------------------ | ------------ | -------------------------------------------------------------- |
+| `lastErrorEvent`   | `String`     | Displays contents of last [ERROR] event                        |
+| `lastWarningEvent` | `String`     | Displays contents of last [WARN] event                         |
+| `lastCustomEvent`  | `String`     | Displays contents of last custom event                         |
+| `errorEvents`      | `Number`     | Displays number of [ERROR] lines matched to search pattern     |
+| `warningEvents`    | `Number`     | Displays number of [WARN] lines matched to search pattern      |
+| `customEvents`     | `Number`     | Displays number of custom lines matched to search pattern      |
+| `logRotated`       | `DateTime`   | Last time when log rotated recognized                          |
+| `newErrorEvent`    | -            | Trigger channel for last [ERROR] line                          |
+| `newWarningEvent`  | -            | Trigger channel for last [ERROR] line                          |
+| `newCustomEvent`   | -            | Trigger channel for last [ERROR] line                          |
+
+## Examples
+
+### example.things
+
+```xtend
+
+logreader:reader:openhablog[ refreshRate=1000, errorPatterns="ERROR+", errorBlacklistingPatterns="annoying error which should ignored|Another annoying error which should ignored" ]
+
+```
+
+### example.items
+
+```xtend
+
+String   logreaderLastError         "Last error [%s]"                                      { channel="logreader:reader:openhablog:lastErrorEvent" }
+String   logreaderLastWarning       "Last warning [%s]"                                    { channel="logreader:reader:openhablog:lastWarningEvent" }
+Number   logreaderErrors            "Error events matched [%d]"                            { channel="logreader:reader:openhablog:errorEvents" }
+Number   logreaderWarnings          "Warning events matched [%d]"                          { channel="logreader:reader:openhablog:warningEvents" }
+DateTime logreaderLogRotated        "Last Log Rotation [%1$tY.%1$tm.%1$te %1$tR]"          { channel="logreader:reader:openhablog:logRotated" } 
+
+```
+
+### example.sitemap
+
+```xtend
+
+sitemap logreader_example label="Example" {
+    Frame label="LogReader" {
+        Text item=logreaderLastError
+        Text item=logreaderLastWarning
+        Text item=logreaderErrors
+        Text item=logreaderWarnings
+        Text item=logreaderLogRotated
+    }
+}
+
+```
+
+### example.rules
+
+```xtend
+rule "LogReader"
+    when
+        Channel "logreader:reader:openhablog:newErrorEvent" triggered
+    then
+        // do something
+    end
+```
+Be carefull when send e.g. email notifications. You could easily send thousand of *spam* emails in short period if e.g. one binding is in error loop.
+
+## Troubleshooting
```

Comment:
Remove this section, there is nothing specific to this binding in here.

Reply:
Done